docs/nodes/credentials/Spotify/README.md

```diff
@@ -0,0 +1,19 @@
+# Spotify
+You can find information about the operations supported by Spotify node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.spotify) page. You can also browse the source code of the node on [Github](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/Spotify).
+
+# Authentication
+
+## Pre-requisites
+
+Create a [Spotify Developer](https://developer.spotify.com/dashboard/login) account.
+
+## Steps
+
+1. Open your Spotify for Developers dashboard.
+2. Create an app.
```

Comment:
Could you please make this more specific? Is there a button you have to click on to do this?